src/components/ConfigureAutomatonWindow.tsx

```diff
@@ -94,18 +109,19 @@ function AlphabetList() {
     <>
       <div className="mt-3 ml-1 mb-1">Input Alphabet</div>
       <div className="divide-y">
-        {tokenWrapperElements}
         <CoreListItem>
           <CoreListItem_Left>
-            <button
-              className="text-blue-500 dark:text-blue-400 flex flex-row items-center"
-              onClick={addTokenToAlphabet}
-            >
-              <BsPlusCircleFill className="mr-1" />
-              Add Token
-            </button>
+            <input
+              className="focus:outline-none bg-transparent grow"
+              type="text"
+              placeholder="q1,q2,q3"
```

Comment:
Tokens can only be one character long, so the placeholder should be as well. Something like `a,b,c`, perhaps?